src/components/PageList.vue

```diff
@@ -174,8 +194,19 @@ export default {
 			'sortBy',
 			'showing',
 			'showTemplates',
+			'allPages',
 		]),
 
+		allpages() {
```

Comment:
Some different wording would be nice to make it easier to differentiate between this and the vuex store getter